frontend/express/public/javascripts/countly/vue/core.js

```diff
@@ -51,7 +51,7 @@
     // @vue/component
     var commonFormattersMixin = {
         methods: {
-            formatTimeAgo: countlyCommon.formatTimeAgo,
+            formatTimeAgo: countlyCommon.formatTimeAgoText,
```

Comment:
It feels like this will break some stuff, but let it break. We need "text" version anyway.

Reply:
actually last i checked formatTimeAgo was not being referenced anywhere, so nothing should break.